pkgs/applications/editors/neovim/default.nix

```diff
@@ -21,16 +22,22 @@ let
         nvim-client luv coxpcall busted luafilesystem penlight inspect
       ]
     ));
+  neovimLuaEnvFB = lua.luaOnBuild.withPackages(ps:
```

Comment:
Before merge will have to take a better look at if both of these env's are needed.

Reply:
Both are necessary, the result has lua libraries in the closure
```
$ nix path-info -rsh ./result
/nix/store/4l7mdm3gmzkbl0z8818s6scpd7iw58s7-linux-headers-6.4                                       	  6.0M
/nix/store/q1y7jmlna5bvs4cwc4vcm3p5wpb2z4rr-aarch64-unknown-linux-gnu-stage-static-gcc-12.3.0-libgcc	460.9K
/nix/store/6vas9fadmxk3af2c8fii3b9qhkiva285-glibc-aarch64-unknown-linux-gnu-2.37-8                  	 39.5M
/nix/store/yjqqgra9bb20zrf7hghyqvj863nyvmmy-glibc-aarch64-unknown-linux-gnu-2.37-8-bin              	  2.9M
/nix/store/26iwgxzps65kll9b098lh5dhsyk4z4ri-glibc-aarch64-unknown-linux-gnu-2.37-8-dev              	  2.2M
/nix/store/2lvlvsr6qf6wgnf1mg1sbzfz931s4fd5-msgpack-c-aarch64-unknown-linux-gnu-6.0.0               	205.4K
/nix/store/qdrwm7fk2x02axzbpc0zfvk394mjjam0-aarch64-unknown-linux-gnu-stage-final-gcc-12.3.0-libgcc 	480.8K
/nix/store/p35nq84lyvz9lz2p3zvvw8pxyvdbcq02-aarch64-unknown-linux-gnu-stage-final-gcc-12.3.0-lib    	  8.0M
/nix/store/2srhcbd1acrcmwkm3ng6d1qw944r1igs-luajit-aarch64-unknown-linux-gnu-2.1.0-2022-10-04       	  2.0M
/nix/store/k9di1jfx4q9cxfihng0g23hj97wcr715-ncurses-aarch64-unknown-linux-gnu-6.4                   	  3.9M
/nix/store/3jrk9v90knd7ip4rf8bmhkgrxdghfm05-libtermkey-aarch64-unknown-linux-gnu-0.22               	112.2K
/nix/store/6f9f93gv00ww0l22j6c4gx6drhibcjf5-luajit2.1-luabitop-1.0.2-3-aarch64-unknown-linux-gnu    	118.1K
/nix/store/7chhcx4k9lfikjpd3x641288pc094z7c-luajit2.1-lpeg-1.0.2-1-aarch64-unknown-linux-gnu        	 86.4K
/nix/store/7n3d15i1i5ilplyn6xq73fyhpgx1gaf5-luajit2.1-mpack-1.0.9-0-aarch64-unknown-linux-gnu       	 76.0K
/nix/store/9y0r7ykn26hsjvnfmxvwchbzln981niq-tree-sitter-aarch64-unknown-linux-gnu-0.20.8            	 12.7M
/nix/store/i9v23n2cgjj533cw662czh3r9hschy10-libuv-aarch64-unknown-linux-gnu-1.46.0                  	236.5K
/nix/store/f1wnky59vh97djz68ybnrvfmf7k7hddf-libluv-aarch64-unknown-linux-gnu-1.44.2-1               	294.4K
/nix/store/hwlmfbx01y9d130v3m1h0yy0ll2if5i7-luajit-aarch64-unknown-linux-gnu-2.1.0-2022-10-04-env   	213.9K
/nix/store/hxm08zycvyg4mbxyzc368g9jhdj70sar-vimdoc-grammar-aarch64-unknown-linux-gnu-neovim-0.9.1   	134.0K
/nix/store/iaxr1x2nl781pvlnnl3154z25006kql4-unibilium-aarch64-unknown-linux-gnu-2.1.1               	285.9K
/nix/store/j708hjl6cx7ap726vjfzrg9iy53cjb6d-libvterm-neovim-aarch64-unknown-linux-gnu-0.3.2         	312.0K
/nix/store/kdd94j72yvhb3rxnc4flb6dry763y3l3-query-grammar-aarch64-unknown-linux-gnu-neovim-0.9.1    	 68.7K
/nix/store/kfxp2ib4yy9jzcpnlxpjvypv94sk5vv4-c-grammar-aarch64-unknown-linux-gnu-neovim-0.9.1        	390.5K
/nix/store/p6g8c5lkz1b4kxkjb6lw8hm30ycab5yv-vim-grammar-aarch64-unknown-linux-gnu-neovim-0.9.1      	  1.2M
/nix/store/x3rnyrc84dzkbx7a8jalr6kfbk3n6rsh-lua-grammar-aarch64-unknown-linux-gnu-neovim-0.9.1      	 69.5K
/nix/store/xxshg2zb8c4pgv88mibm4z37f1laps37-neovim-unwrapped-aarch64-unknown-linux-gnu-0.9.1        	 25.3M
```

they're for the correct system so no issue there